aps/lib/aps.ex

```diff
@@ -1,5 +1,86 @@
 defmodule InfinityAPS do
   @moduledoc """
-  Documentation for InfinityAPS.
+  Functions used throughout `aps`.
   """
+
+  @doc """
+  Returns the environment variable of the loop directory,
+  where oref0 can find and save the json files needed to loop.
+  """
+  def loop_directory do
+    Application.get_env(:aps, :loop_directory)
+  end
+
+  @doc """
+  Returns the environment variable of the configuration file.
+  """
+  def configuration_file do
+    Application.get_env(:aps, :configuration_file)
+  end
+
+  @doc """
+  Returns the environment variable of the node_modules directory.
+
+  It joins any `file_path` given.
+  """
+  def node_modules_directory(file_path \\ "") do
+    Path.join(Application.get_env(:aps, :node_modules_directory), file_path)
+  end
+
+  @doc """
+  Returns the environment variable of the module responsible
+  for pump communication.
+  """
+  def pump do
+    Application.get_env(:pummpcomm, :pump)
+  end
+
+  @doc """
+  Returns the environment variable of the module responsible
+  for cgm communication.
+  """
+  def cgm do
+    Application.get_env(:pummpcomm, :cgm)
+  end
+
+  @doc """
+  Returns true if the input is a sensor_glucose_value.
+  Otherwise, it returns false.
+
+  Used for sgv filtering with Enum.filter\1.
```

Comment:
I think this should be a forward slash between the function name and arity

Reply:
Indeed!!